### DISCUSSION TITLE
Add support for rhel8 based stand-alone deployment

### DISCUSSION THREAD

**Discussion by nmlsdev** [mediacms-io/mediacms#780] [Feature Requests]
Hello,
     I have gotten mediacms deployed in a stand-alone configuration on rocky 8, and currently have an early version of an install.sh script to help automate the process a bit. I was wondering if anyone in the project would be interested in expanding the capabilities of this project to allow for others with my use-case. I'd be happy to help maintain this effort, and would be open to discussing it further with anyone currently involved in maintaining it.
v/r,
nmls

**Comment by mgogoulos** [comment]
Hi @nmlsdev , at the very least consider contributing the file here (if it's a single file), this will definitely help any others using the same environment (RHEL based distros).

Regards

**Reply by nmlsdev** [reply]
I'd love to, there's still some user management and postgres configuration I need to automate, but once it's done I suppose I'd create a new pull request? I've worked with git a while, but never contributed to anything on github before so I'm not 100% on how to go about that.

Thank you!

**Reply by mgogoulos** [reply]
A pull request would be fine, and I can add comments there before we merge it.

**Reply by nmlsdev** [reply]
Submitted pull request #792
